components/browser/session/build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
     // We expose this as API because we are using Observable in our public API and do not want every
     // consumer to have to manually import "support-base".
     api project(':support-base')
+    api project(':concept-fetch')
```

Comment:
Why does `browser-session` need this?

Reply:
Thanks for catching this, this was by mistake.